origin-discovery/listener/listener.js

```diff
@@ -488,14 +488,24 @@ async function handleLog(log, rule, contractVersion, context) {
 
   if (context.config.webhook) {
     console.log('\n-- WEBHOOK to ' + context.config.webhook + ' --\n')
-    await withRetrys(async () => {
-      await postToWebhook(context.config.webhook, json)
-    })
+    try {
+      await withRetrys(async () => {
+        await postToWebhook(context.config.webhook, json)
+      }, exitOnError=false)
+    } catch(e) {
+      console.log(`Skipping webhook for ${logDetails}`)
```

Comment:
What is the consequence of this? I'm thinking the worst thing that can happen is that people don't receive notifications, but nothing worse than that?

Reply:
Yes, exactly. I think it is better to not send those notifications vs entirely stopping the event-listener which will impact indexing and freshness of the data displayed in the DApp (since soon we'll rely on back-end data for powering DApp).
Ideally we would log a stat here that would trigger alerting so that we can quickly take a look.